Copy prebuild artefacts into sourcetree and add flags for parallel builds in VS.

danielfenner opened the pull request:
Ugly trick to get the CI build in minutes instead of hours. 

This requires me to update the vm building this when we do nontrivial changes to the dependecies. If this hurts at some point I'll look into properly caching this in some persistent storage that gets updated by the build system.

pierricgimmig commented:
I think it's fine for now.  It would be great if we could eventually ship a set of prebuilt dependencies, possibly through that vcpkg export feature.  That would be a huge time saver for anyone compiling for the first time.

danielfenner commented:
The initial proposal did not work. There are a bunch of other things that need to be copied. Fixed that.

I also looked into using "vcpkg export" but didn't get it to work. Some packages need their buildtree around to work properly.

Added a compile switch to enable parallel compilation in VS.
